.gitpod.yml

```diff
@@ -15,6 +15,19 @@ tasks:
       cd "$GITPOD_REPO_ROOT"
       cp .bashrc.d/100-firefox-dev ~/.bashrc.d/
       source ~/.bashrc.d/100-firefox-dev
+      
+      # Try server: if both TRY_USERNAME and TRY_SSH_KEY environment variables are present,
+      # configure the credentials for pushing to try.
+      #
+      # (This is not required, but useful if you have Try access and want to push builds to try.)
+      #
+      # NB: when defining the TRY_SSH_KEY env var variable in GitPod web UI, replace newlines by "\n".
+      function configure_try_credentials() {
+        echo -e "Host hg.mozilla.org\n  User ${TRY_USERNAME}" >> ~/.ssh/config
+        echo -e "$TRY_SSH_KEY" > ~/.ssh/id_rsa
+        chmod 600 ~/.ssh/id_rsa
+      }
+      [[ ! -z "${TRY_USERNAME}" ]] && [[ ! -z "${TRY_SSH_KEY}" ]] && configure_try_credentials
```

Comment:
What do you think about prefixing each env var with `FIREFOX_` to avoid conflicts? I think env vars are per user account and not necessarily per workspace.

Reply:
It seems Gitpod's env vars can be scoped by repository if needed - but prefixing everything by 'FIREFOX_' seems sensible to me 👍